src/d/component.spec.ts

```diff
@@ -1,25 +1,31 @@
 import { Component, User } from './component';
 
-const mockData: User = { id: 'foo', name: 'bar', age: 40 };
-
 describe('Component', () => {
     let component: Component;
     beforeEach(() => {
         component = new Component();
-        component.setUser(mockData);
     });
 
+    function mockUser(): User {
+        return <User>{ id: 'foo', name: 'bar', age: 40 };
+    }
+
     it('should return age as undefined if no user', () => {
         expect(component.age).toBeUndefined();
     });
 
     it('should correctly identify the user age', () => {
+        const mockData = mockUser();
+        component.setUser(mockData);
         expect(component.age).toEqual(mockData.age);
     });
 
     it('should increment the age of the user', () => {
+        const mockData = mockUser();
+        const mockAge = mockData.age;
+        component.setUser(mockData);
         component.incrementAge();
-        expect(component.age).toEqual(41);
+        expect(component.age).toEqual(mockAge + 1);
```

Comment:
saves declaring it above, but just preference so no requirement to change
```suggestion
        expect(component.age).toEqual(mockData.age + 1);
```